File: Selnium script for testing/Script with Python/test_scenario_38.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_38():
    """
    Dummy test run for scenario 38
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_38")

    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.ID, "user_name").send_keys("test_data_583")
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    time.sleep(2)

    driver.quit()

if __name__ == "__main__":
    execute_test_38()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario38.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario38 {
    public static void executeTest38() {
        /*
         * Dummy test run for scenario 38
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_38");

        try {
            driver.findElement(By.linkText("Log Out")).click();
            if (!driver.findElement(By.id("user_name")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.name("password_field")).click();
            driver.findElement(By.className("nav-item")).sendKeys("test_data_983");
            driver.findElement(By.className("nav-item")).click();
            driver.findElement(By.xpath("//button[@type='submit']")).click();
            Thread.sleep(2000);
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.cssSelector(".promo-code")).sendKeys("test_data_790");
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_735");
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest38();
    }
}
